### ISSUE TITLE
JAWS/NVDA is announcing hex codes instead of proper color names while navigating through colors in 'Select color' popup

### ISSUE THREAD

**Issue by sryarama** [ckeditor/ckeditor4#4482]
## Type of report

Bug

## Provide detailed reproduction steps (if any)
1) Turn on JAWS/NVDA.
2) Open CKEDITOR demo site https://ckeditor.com/ckeditor-4/demo/
3) Tab till Editor text area and press 'Alt+F10' to navigate to toolbar.
4) Tab till 'Background color' button in tool bar and press enter and select 'More colors..'
5) Verify whether screen reader is announcing the color names or not while navigating through colors in 'Select color' popup.

### Expected result
Screen reader should announce proper color names along with hex code values while navigating through colors in 'Select color' popup. 

### Actual result
JAWS/NVDA is announcing hex codes instead of proper color names while navigating through colors in 'Select color' popup in email editor text area.

### User Impact
Screen reader users may find it difficult to understand the focused color when screen reader announces hex codes in 'Select color' popup. 

### Ask
Is it possible to show color names? if not do you have any future tasks to achieve this.

## Other details

* Browser: Google Chrome Version 87.0.4280.66 (Official Build) (64-bit)
* OS: Windows 10
* CKEditor version: 4
* Screen Reader: JAWS and NVDA.

**Comment by Comandeer** [comment]
It's rather a feature request, as we do not provide color names at all at the moment. Probably we could introduce mapping hex values to [CSS Color module color keywords](https://drafts.csswg.org/css-color/#named-colors).

> if not do you have any future tasks to achieve this.

Unfortunately, no. I'll transform this issue into a feature request.

**Comment by sryarama** [comment]
Hi Team, is there any update on this?

**Comment by jacekbogdanski** [comment]
Hey @sryarama, so far we have no progress on this issue due to other priorities in CKEditor 4.

If you looking for a way to push this issue forward, you may contact our [support team](https://ckeditor.com/contact/) to see if there are any options to speed things up.
